packages/rpc-core/src/rpc-methods/__tests__/get-account-info-test.ts

```diff
@@ -34,11 +34,11 @@ describe('getAccountInfo', () => {
 
                 await expect(accountInfoPromise).resolves.toMatchObject({
                     value: expect.objectContaining({
-                        data: expect.any(String),
-                        executable: expect.any(Boolean),
-                        lamports: expect.any(BigInt),
-                        owner: expect.any(String),
-                        rentEpoch: expect.any(BigInt),
+                        data: '2Uw1bpnsXxu3e',
+                        executable: false,
+                        lamports: 5000000n,
+                        owner: '11111111111111111111111111111111',
+                        rentEpoch: 18446744073709551616n, // TODO: This number loses precision
```

Comment:
In #1534, I changed CI to run against `stable` (ie. 1.14.24 as of now) which I think emits `0` for this value.

1. If you're not running `stable` locally when you run these tests, you should nuke your validator and re-run `pnpm test:live-with-test-validator:setup`
2. I should figure out a way to install the `stable` validator _locally_ (ie. in the repo, rather than globally) so that you don't have to nuke your global copy

Reply:
The second bullet shouldn't be too bad. I can mess around with it.
I'm thinking something like building it from source and referencing the compiled Rust binary directly ie:

```shell
target/debug/solana-test-validator
```

Reply:
I _think_ the install script has a `--data-dir` option, but I haven't tried it yet. Give it a shot? We should just install into like `./.solana/` and then add `.solana/` to the `.gitignore`.

https://github.com/solana-labs/solana/blob/4d452fc5e9fd465c50b2404354bbc5d84a30fbcb/install/solana-install-init.sh#L38

Reply:
In any case we probably don't want to build from source because that would be slow and make it difficult for people to contribute to web3.js if they don't have the whole compilation toolchain installed.

Reply:
So I just finally circled back to this and I realized this _doesn't_ emit a 0, and maybe it's because our fixture has a value for rentEpoch?

Reply:
> I _think_ the install script has a `--data-dir` option, but I haven't tried it yet. Give it a shot? We should just install into like `./.solana/` and then add `.solana/` to the `.gitignore`.

This worked like a charm